specs/time_series_api.adoc

```diff
@@ -107,6 +107,10 @@ custom collectd plugins rather than the collectd provided ones
 
 The tendrl-api interfaces to the apis mentioned above under the section
 "Proposed change" and detailed below under "Notifications/Monitoring impact".
+Making call's to the monitoring app is via a http client programatically. The
+tendlr-api will define its own url's, somewhat similar to what API's from the
+monitoring app. We need to take this approach since we need to have one point
+of interaction.
```

Comment:
I  guess what you mentioned here was the APIs exposed by monitoring app. Its good to mention them as well here becuse, those are the one's exposed to the end user